samples/nodejs-typescript-food-catalog/teamsapp.yml

```diff
@@ -7,6 +7,24 @@ environmentFolderPath: ./env
 
 provision:
 
+  - uses: aadApp/create
+    with:
+      name: ${{APP_NAME}}-${{TEAMSFX_ENV}}
+      generateClientSecret: true
+      signInAudience: AzureADMultipleOrgs
```

Comment:
This brings up a good point: if we want to use MSI, which would be recommended, it would only work with single-tenant apps. Let's see if we could make two variants of this (single- and multitenant, through separate .env files)